### DISCUSSION TITLE
Status of development

### DISCUSSION THREAD

**Discussion by paulheu** [DorianTheGrey/DU-DamageReport#28] [General]
What is the current status of development for this project?
Would it be possible to share the non minified source code for the last available version, so others can continue work on this?

**Comment by DorianTheGrey** [comment]
Hi there, this project is parked for good as Novaquark and their terrible way of updating and communicating has annoyed me beyond anything I have ever experienced from any other developer. Invested thousands of hours into their project and then got pooped on.

By now, several people have already taken the code, have in fact stolen it by changing the original copyright in the code and on the screens and have apparently partially worked on it. So, I am done with this community. Please refer to these people, as this project will be archived soon just as my own DU accounts were unsubscribed months ago.

**Reply by paulheu** [reply]
I get your frustration and certainly have no love for NQ for similar reasons as you describe. But I do like the game and while I understand you just want to burn it all and that others may have ripped your code and run with it is not helping, I'd rather give credit where due and not get stuck with hacks ;)

**Reply by DorianTheGrey** [reply]
Sure, if I can help out: send me a mail to jan.seidler.ch@gmail.com and I can send you the latest non-minified version. :)

**Reply by lambolandersnft** [reply]
Hey Dorian, did you build this in Visual Studio? What plugins, set up did you have to write this?

**Reply by DorianTheGrey** [reply]
Hi there, mate. I have developed this using only the Sublime editor, and well, of course DU. :) I know, e.g. Archeago has a much more advanced setup, so you might want to consult with him. I focused on LUA and SVG and rather tested live in DU. I didn't even use any local tool for minimizing, there's plenty of sites out there doing it in a second with two copy & pastes.

**Reply by PanicRide** [reply]
I'm interested in contributing to keep this alive as well. Are you interested in opening the original source under this license?

**Reply by lambolandersnft** [reply]
Thanks for your reply appreciate it - I was keen to get the script running and updated the references but it was booting to a blank screen. I really wished this still worked. It was definitely the best.

**Reply by PanicRide** [reply]
It's likely because it still renders the screens in the old html format which is now disabled in the game client by default for performance reasons. If you enable it and restart the game, the screens shouldn't appear blank for you anymore.
